src/rules/attribute-value-entities.ts

```diff
@@ -65,7 +65,7 @@ const rule: Rule.RuleModule = {
                 );
                 const rawValue = analyzer.getRawAttributeValue(element, attr);
 
-                if (!loc || !rawValue?.value) {
+                if (!loc || !(rawValue && rawValue.value)) {
```

Comment:
just to be a bit clearer can we go with `if (!loc || !rawValue || !rawValue.value)`

Reply:
Yeah thought about that as well. Can do